yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -405,7 +405,7 @@ func (ar *AssessmentReport) CalculateSizeDetails() (SizeDetails, error) {
 			} else {
 				details.TotalTableSize += utils.SafeDereferenceInt64(stat.SizeInBytes)
 				details.TotalTableRowCount += utils.SafeDereferenceInt64(stat.RowCount)
-				if slices.Contains(colocatedTables, fmt.Sprintf("%s.%s", stat.SchemaName, stat.ObjectName)) {
+				if slices.Contains(colocatedTables, stat.ObjectName) {
```

Comment:
@makalaaneesh why are we changing this? At this time there will be fully qualified names present.

I had added a function just before generating report for removing the schema names `postProcessingOfAssessmentReport()`

Reply:
ignore my comment, it will be required

Reply:
At this time when the function is called, postprocessing is already done. (because this is done at the last when we want to send the event to callhome

Reply:
Right